
# HG changeset patch
# User Stuart Rackham <srackham@methods.co.nz>
# Date 1215931224 -43200
# Node ID 351cdc59164fc8adc5c5d95f7b1fdb2db46f3893
# Parent 3af946545ba8f2f91271d09e81be890c5651057d
< and > characters excluded from URL macros so the URL is not incorrectly recognized due to prior substitutions.

--- a/asciidoc.conf	Sun Jul 13 18:03:54 2008 +1200
+++ b/asciidoc.conf	Sun Jul 13 18:40:24 2008 +1200
@@ -173,11 +173,11 @@
 (?su)[\\]?(?P<name>http|https|ftp|file|mailto|callto|image|link):(?P<target>\S*?)(\[(?P<attrlist>.*?)\])=
 
 # These URL types don't require any special attribute list formatting.
-(?su)(?<!\S)[\\]?(?P<name>http|https|ftp|file):(?P<target>//\S*[\w/])=
+(?su)(?<!\S)[\\]?(?P<name>http|https|ftp|file):(?P<target>//[^ \t<>]*[\w/])=
 # Allow a leading parenthesis.
-(?su)(?<\=\()[\\]?(?P<name>http|https|ftp|file):(?P<target>//\S*[\w/])=
+(?su)(?<\=\()[\\]?(?P<name>http|https|ftp|file):(?P<target>//[^ \t<>]*[\w/])=
 # Allow <> brackets.
-(?su)[\\]?&lt;(?P<name>http|https|ftp|file):(?P<target>//\S*[\w/])&gt;=
+(?su)[\\]?&lt;(?P<name>http|https|ftp|file):(?P<target>//[^ \t<>]*[\w/])&gt;=
 
 # Email addresses don't require special attribute list formatting.
 # The before ">: and after "< character exclusions stop multiple substitution.

